py/src/ai/h2o/sparkling/ml/params/H2OCommonParams.py

```diff
@@ -61,42 +42,18 @@ class H2OCommonParams(H2OBaseMOJOParams):
     ##
     # Getters
     ##
-    def getFoldCol(self):
-        return self.getOrDefault(self.foldCol)
-
-    def getWeightCol(self):
-        return self.getOrDefault(self.weightCol)
-
     def getSplitRatio(self):
         return self.getOrDefault(self.splitRatio)
 
-    def getSeed(self):
-        return self.getOrDefault(self.seed)
-
-    def getNfolds(self):
-        return self.getOrDefault(self.nfolds)
-
     def getColumnsToCategorical(self):
         return self.getOrDefault(self.columnsToCategorical)
 
     ##
     # Setters
     ##
-    def setFoldCol(self, value):
-        return self._set(foldCol=value)
-
-    def setWeightCol(self, value):
-        return self._set(weightCol=value)
-
     def setSplitRatio(self, value):
         return self._set(splitRatio=value)
 
-    def setSeed(self, value):
```

Comment:
Method `setWithContributionsCol` is missing across the code, but I guess that is because this branch is not on top of master?

Reply:
I've rebased to master.

Reply:
(see [line 92](https://github.com/h2oai/sparkling-water/pull/2159/files#diff-0c9f45aaa9f977bf9890ad82d269e9f8R92)